package.json

```diff
@@ -48,6 +48,6 @@
     "opal-compiler": "1.0.7"
```

Comment:
I think we need to update the `opal-compiler` to `1.0.13` because the code produced by Opal is not always compatible between versions.